docs/creating-a-runtime-check.md

```diff
@@ -0,0 +1,209 @@
+# Creating a Runtime Check
+
+Runtime checks are used to perform tests by executing the plugin's code.
+
+The Plugin Checker uses a number of interfaces to build out runtime checks.
+
+The Plugin Checker provides the `Runtime_Check` interface, which is used to identify a runtime check. This interface does not contain any methods but serves as a marker for runtime checks.
+
+```php
+use WordPress\Plugin_Check\Checker\Check_Result;
+use WordPress\Plugin_Check\Checker\Runtime_Check;
+use WordPress\Plugin_Check\Traits\Stable_Check;
+
+class Custom_Check extends Runtime_Check {
```

Comment:
See my feedback on the static check code example.
```suggestion
class My_Custom_Check implements Runtime_Check {
```

Reply:
Updated